Symfony/CS/Tests/Fixer/Contrib/MethodArgumentDefaultValueFixerTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Contrib;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+class MethodArgumentDefaultValueFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideExamples
+     *
+     * @param string $expected
+     * @param string $input
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+    /**
+     * @return array
+     */
+    public function provideExamples()
+    {
+        $input1 = <<<'EOT'
+                    <?php
+                        public function eFunction($foo, $bar, \SplFileInfo $baz, $x = 'default';
+
+                        public function fFunction($foo, $bar = 'removedValue', \SplFileInfo $baz, $x = 'default');
+EOT;
+
+        $expected1 = <<<'EOT'
+                    <?php
+                        public function eFunction($foo, $bar, \SplFileInfo $baz, $x = 'default';
+
+                        public function fFunction($foo, $bar, \SplFileInfo $baz, $x = 'default');
+EOT;
+
+        return array(
```

Comment:
Ok, will add one, but probably tomorrow. Am curious to see if it would work on that, don't see a problem yet though.